tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -1647,6 +1647,7 @@ def test_volume_data_assistant_result_serialization(
         bobby_volume_data_assistant_result.to_json_dict()
         == volume_data_assistant_result_as_dict
     )
+    assert len(bobby_volume_data_assistant_result.profiler_config.rules) == 2
```

Comment:
Why is this here? I don't think it fits the scope of the PR right?

Reply:
@cdkini Good catch -- I felt to add this as a nice test for the symmetry with the onboarding data assistant test of the same kind.  Thanks!